src/window_background/labels.js

```diff
@@ -25,13 +25,12 @@ import {
 import actionLog from "./actionLog";
 import addCustomDeck from "./addCustomDeck";
 import { createDraft, createMatch, completeMatch } from "./data";
-
-var logLanguage = "English";
+import * as httpApi from "./http-api";
```

Comment:
Is there a reason to use a namespace import?  The identifiers are all prefixed with `http`.

```suggestion
import {
  httpSetDraft,
  httpSetMatch,
  httpSetMythicRank,
  httpSetSeasonal,
  httpSubmitCourse,
  httpTournamentCheck
} from "./http-api";
```

Reply:
No reason really, its a leftover from when the file was created and used as exported functions.